examples/task.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       name: NAMESPACE
       description: Namespace of the application under test
   steps:
-    - image: quay.io/redhat-appstudio/hacbs-test:latest
+    - image: quay.io/konflux-ci/konflux-test:latest
```

Comment:
Just a question, do we know when those repos in quay will be migrated also?

Reply:
uff this is wrong, this shouldn't be migrated

Reply:
fixed to `redhat-appstudio/konflux-test` for images we are keeping old org, only image name changed